packages/auth-server/__tests__/globalSetup.js

```diff
@@ -0,0 +1,23 @@
+import { exec } from 'child_process';
+
+import { create } from './external-dependencies/database/test-database.js';
+import { MockedExternalApis } from './external-dependencies/rest-apis/mocked-external-apis.js';
+
+function run(cmd) {
+	return new Promise((resolve, reject) => {
+		exec(cmd, (err, stdout) => {
+			if (err) {
+				reject(err);
+			} else {
+				resolve(stdout);
+			}
+		});
+	});
+}
+
+export default async () => {
```

Comment:
slight difference between appeals api here, in that setup/teardown is done once for notify mock, rather than on every test